openapi/mt_openapi_spec_v1.yaml

```diff
@@ -13981,12 +14072,17 @@ components:
             default. Learn more about changing your settings at https://docs.moderntreasury.com/reference/personally-identifiable-information.
         counterparty_id:
           type: string
+          nullable: true
           description: The ID for the counterparty for this transaction line item.
         expected_payment_id:
           type: string
+          nullable: true
           description: The ID of the reconciled Expected Payment, otherwise `null`.
+        transaction_id:
+          type: string
+          description: 'Translation missing: en.openapi.descriptions.transaction_line_item.schema.transaction_id'
```

Comment:
don't merge - this needs to be fixed

Reply:
@chen-annie you're right i'll put up a PR!